user/views.py

```diff
@@ -1,3 +1,121 @@
-from django.shortcuts import render
+import json,      re, bcrypt, jwt
+from django.http  import JsonResponse
+from django.views import View
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from lineamigos.settings    import SECRET_KEY, ALGORITHM
+from .models import User, PhoneNumber, Gender
+from .utils  import SignInConfirm
```

Comment:
```suggestion
import json, re, bcrypt, jwt

from django.http            import JsonResponse
from django.views           import View
from django.core.exceptions import ObjectDoesNotExist, ValidationError

from lineamigos.settings import SECRET_KEY, ALGORITHM

from .models import User, PhoneNumber, Gender
from .utils  import SignInConfirm
```
이런식으로 import 문이 많을 경우 분류해서 나눠주시면 가독성이 좋아집니다.

Reply:
리뷰해주신 부분 반영해서 수정했습니다!